src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/CreditManager.cs

```diff
@@ -36,6 +36,50 @@ private object SyncObject
             get => this;
         }
 
+        public int RequestCredit(int amount, CancellationToken cancellationToken)
```

Comment:
This is HTTP/2 code, I thought this PR should only affect HTTP/1 path?

Reply:
Removed, it wasn't even used.